webapp/channels/src/components/widgets/modals/components/save_changes_panel.tsx

```diff
@@ -1,32 +1,80 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
+import classNames from 'classnames';
 import React from 'react';
 import {FormattedMessage} from 'react-intl';
 
-import './save_changes_panel.scss';
 import {AlertCircleOutlineIcon} from '@mattermost/compass-icons/components';
 
+import './save_changes_panel.scss';
+
+export type SaveChangesPanelState = 'saving' | 'saved' | 'error';
```

Comment:
I feel the `saving` state is misleading. Shouldn't be more like `editing`?

Reply:
Good point, `editing` is better reflecting the state